Sources/Stencil/ColorsContext.swift

```diff
@@ -27,14 +27,23 @@ extension ColorsFileParser {
 
       return [
         "name": name,
-        "rgba": String(hexChars[0..<8]),
-        "rgb": String(hexChars[0..<6]),
         "red": comps[0],
         "green": comps[1],
         "blue": comps[2],
-        "alpha": comps[3]
+        "alpha": comps[3],
+        
+        // NOTE: This is a deprecated variable
+		"rgba": String(hexChars[0..<8]),
+		"rgb": String(hexChars[0..<6])
       ]
     }).sorted { $0["name"] ?? "" < $1["name"] ?? "" }
-    return ["enumName": enumName, "colors": colorMap]
+    
+    return [
+      "colors": colorMap,
+      "param": ["enumName": enumName],
```

Comment:
The way SwiftGen's CLI inject command line `--param x=y` parameters means that this will be overridden by extra params, right?